src/libraries/System.Private.CoreLib/src/System/OperatingSystem.cs

```diff
@@ -331,17 +331,16 @@ private static bool IsOSVersionAtLeast(int major, int minor, int build, int revi
             {
                 return current.Major > major;
             }
-            if (current.Minor != minor)
+            if (current.Minor >= 0 && current.Minor != minor)
```

Comment:
This one is not needed, since `new Version` and `Version.Parse` require major and minor versions at minimum. [sharplab](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEUCuA7AHwAEAmARgFgAoaosgTgAo8YB3AAgDUYoBnASwh5GZNCQCU4gNzVqAejnswACwCGeAOYx2GCOzIA6ErQaNufQXgMAFVXxiMARGUeSZVIA=)

Reply:
Thanks